packages/openapi-generator/src/wrapper-files/index-file.ts

```diff
@@ -1,13 +1,21 @@
-import { codeBlock } from '@sap-cloud-sdk/util';
+import { codeBlock, first } from '@sap-cloud-sdk/util';
 
 /**
  * @experimental This API is experimental and might change in newer versions. Use with caution.
  * Generate the index file to expose the SAP Cloud SDK API and models.
+ * @param apis The generated api files.
  * @returns The index file contents.
  */
-export function indexFile(): string {
+export function indexFile(apis: string[]): string {
   return codeBlock`
 export * from './openapi/model';
-export * from './api';
+${apis
+  .filter(api => api.endsWith('.ts'))
```

Comment:
can it be that non ts files are in the list? From the usage in generator.ts it does not look like it.

Reply:
I'll refactor it